recipes/machinevision-toolbox-python/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "machinevision-toolbox-python" %}
+{% set version = "0.9.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/machinevision-toolbox-python-{{ version }}.tar.gz
+    sha256: b3b70f230246bf7ca22ffda6c61bfa99c6c636c91d44bb2f10617ed7cab73fab
+  - path: RELEASE
```

Comment:
i wonder if it would be easier to create this file during build time? You could use something like `echo "${PKG_VERSION}" > RELEASE`